packages/pharos-site/src/pages/index.tsx

```diff
@@ -36,233 +34,229 @@ const IndexPage: FC = () => {
     typeof window !== `undefined` ? require('@ithaka/pharos/lib/react-components') : null;
 
   useEffect(() => {
-    const { PharosHeading, PharosLink, PharosIcon, PharosButton } = Pharos;
+    const { PharosHeading, PharosLink, PharosIcon, PharosButton, PharosLayout } = Pharos;
 
     const content = (
       <>
-        <div className={hero}>
-          <div className={hero__text}>
-            <h1 className={hero__heading}>
-              The Pharos design system is our guiding light toward creating <i>cohesive, </i>
-              <i>supportive, </i> and <i>beautiful</i> experiences for the{' '}
-              <i>intellectually curious</i>.
-            </h1>
-            <PharosHeading level="2" preset="1--bold" className={heading__status}>
-              Get a detailed breakdown of components and the status of their implementation
-            </PharosHeading>
-            <PharosLink
-              className={link__status}
-              href="components/component-status"
-              onBackground
-              flex
-              data-sc="link:component status hero"
-              onClick={handleLinkClick}
-            >
-              Check out the component status page
-              <PharosIcon name="arrow-right" className={icon__arrow}></PharosIcon>
-            </PharosLink>
-          </div>
-        </div>
+        <PharosLayout preset="1-col--sidenav-comfy" rowGap="0" className={hero}>
+          <h1 className={hero__heading}>
+            Pharos, JSTOR&apos;s design system, guides us to create <i>cohesive, </i>
```

Comment:
Did this wording change intentionally? I might've missed the discussion point.

Reply:
Yeah @kelseytrabue updated it